### ISSUE TITLE
The flashing of the forum

### ISSUE THREAD

**Issue by dariaemacs** [rafalp/Misago#796]
I was enjoying while I was catching this bug. 
Last version Misago.
djangojs.po: 1764

```
#: static/misago/js/misago.js:13 static/misago/js/misago.js:16
msgid "%(replies)s reply"
msgid_plural "%(replies)s replies"
msgstr[0] "%(replies)s bla-bla"
msgstr[1] "%(replies)s blas-blas"
```
After msgstr initialization the categories disappear and the forum begins to blink. This happens with msgid "%(replies)s reply" only.

![screenshot from 2017-04-15 07 40 04](https://cloud.githubusercontent.com/assets/22542295/25060811/f98a5d7e-21ae-11e7-96f8-c1150dbea500.png)
![screenshot from 2017-04-15 07 40 08](https://cloud.githubusercontent.com/assets/22542295/25060813/008fdb76-21af-11e7-8fe8-4cbeaf3b5a27.png)

**Comment by dariaemacs** [comment]
The same situation in "Moderator View" (/users/moderator/) . But I don't know why. :(((( (cry)

![screenshot from 2017-04-15 08 47 50](https://cloud.githubusercontent.com/assets/22542295/25061143/647f1e9a-21b8-11e7-847c-627742b96975.png)

![screenshot from 2017-04-15 08 47 51](https://cloud.githubusercontent.com/assets/22542295/25061144/6adadf7c-21b8-11e7-9515-e0e5f9cb5b1d.png)

**Comment by rafalp** [comment]
Looks like this happens because Russian language requires four pluralization forms to work correctly, and you've filled in only two. I have already warned you about this being trap in makemessages workings, in second part of [this message](https://github.com/rafalp/Misago/issues/786#issuecomment-291960896). ;)

**Comment by rafalp** [comment]
Yup. You are seeing errors because Misago tries to get third form for pluralizations, but gets `undefined` instead. This results in forum crashing because it expects to call `string.replace` but runs `undefined.replace`, which is an error in javascript.

I've edited `po` files to contain one more form (i've skipped last form used for decimals, because its nonsensical in how misago uses pluralizations), so your example looks like this:

```
msgid "%(replies)s reply"
msgid_plural "%(replies)s replies"
msgstr[0] "%(replies)s form 0"
msgstr[1] "%(replies)s form 1"
msgstr[2] "%(replies)s form 2"
```

This fixes the crash for me:

<img width="566" alt="zrzut ekranu 2017-04-15 o 18 48 17" src="https://cloud.githubusercontent.com/assets/750553/25065236/7091eefc-220c-11e7-8ea9-3fd60e3ba5b9.png">

Long story short, you'll need to go trough django-generated po files and make sure no `msgstr[2]` keys are missing.

**Comment by rafalp** [comment]
On second note, this is makes good reason to use some translation assisting translation software, like transifex, because it spots lackings like this and fixes them for you ;)

**Comment by dariaemacs** [comment]
Thank you, Rafał! Sorry for troubling you with it!
I didn't know that. I used to see console bugs :), not strange behaviour of the web page. 
I'm confused a little :)...
Thanks again!

**Comment by dariaemacs** [comment]
Your project is so beautiful, modern and comfortable in use. 
I appreciate you for Misago! You are super programmer! :)
